lang/cs/Org.Apache.REEF.sln

```diff
@@ -76,144 +76,263 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Org.Apache.REEF.FatNuGet",
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Org.Apache.REEF.Bridge.CLR", "Org.Apache.REEF.Bridge.CLR\Org.Apache.REEF.Bridge.CLR.csproj", "{81575917-C673-4E6F-9CF1-0DD35FC5E717}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Org.Apache.REEF.Bridge.Core.Common", "Org.Apache.REEF.Bridge.Core.Common\Org.Apache.REEF.Bridge.Core.Common.csproj", "{5961DBD3-9A65-4B9C-B9D2-7139954F3163}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Org.Apache.REEF.Bridge.Core.HelloREEF", "Org.Apache.REEF.Bridge.Core.HelloREEF\Org.Apache.REEF.Bridge.Core.HelloREEF.csproj", "{5A3AD8DA-D876-4EA5-A96D-16E099C87BE4}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Org.Apache.REEF.Bridge.Core.Grpc", "Org.Apache.REEF.Bridge.Core.Grpc\Org.Apache.REEF.Bridge.Core.Grpc.csproj", "{60F765D6-4BC9-45A0-8EED-ECEB5E56EDAD}"
+	ProjectSection(ProjectDependencies) = postProject
+		{448A0D5F-A8DE-4CAA-8224-0A9A63A9392E} = {448A0D5F-A8DE-4CAA-8224-0A9A63A9392E}
+	EndProjectSection
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Org.Apache.REEF.Bridge.Core.Driver", "Org.Apache.REEF.Bridge.Core.Driver\Org.Apache.REEF.Bridge.Core.Driver.csproj", "{848458F1-0E94-46A2-A35D-03F96286182A}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Org.Apache.REEF.Bridge.Core.Tests", "Org.Apache.REEF.Bridge.Core.Tests\Org.Apache.REEF.Bridge.Core.Tests.csproj", "{F5A9F4B6-9CFE-46A2-96FE-1C926445D88B}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Org.Apache.REEF.Bridge.Core.Proto", "Org.Apache.REEF.Bridge.Core.Proto\Org.Apache.REEF.Bridge.Core.Proto.csproj", "{CA34E32B-0028-484C-B71C-75FE38036F08}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU

```

Comment:
So VS insists that we should have an Any CPU config in the solution. I usually delete them before checking in. Do you think these should stay?